### ISSUE TITLE
1TDEREVO

### ISSUE THREAD

**Issue by bsopilnyak-eng** [bsopilnyak-eng/Module1IAD#1]
import pandas as pd
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split
from sklearn.tree import DecisionTreeClassifier, plot_tree
import matplotlib.pyplot as plt

data = load_iris()
X = pd.DataFrame(data.data, columns=data.feature_names)
y = pd.Series(data.target)

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=42)
model = DecisionTreeClassifier(random_state=42, max_depth=3)
model.fit(X_train, y_train)

plt.figure(figsize=(12,6))
plot_tree(model, feature_names=data.feature_names, class_names=data.target_names, filled=True)
plt.show()

print("Accuracy:", model.score(X_test, y_test))
